src/promptflow/promptflow/_sdk/_service/apis/collector.py

```diff
@@ -29,14 +29,21 @@
 from promptflow._utils.thread_utils import ThreadWithContextVars
 
 
-def trace_collector(get_created_by_info_with_cache: Callable, logger: logging.Logger):
+def trace_collector(
+    get_created_by_info_with_cache: Callable,
+    logger: logging.Logger,
+    cloud_trace_only: bool = False,
+    credential: object = None,
+):
     """
     This function is target to be reused in other places, so pass in get_created_by_info_with_cache and logger to avoid
     app related dependencies.
 
     Args:
         get_created_by_info_with_cache (Callable): A function that retrieves information about the creator of the trace.
         logger (logging.Logger): The logger object used for logging.
+        cloud_trace_only (bool): If True, only write trace to cosmosdb and skip local trace. Default is False.
+        credential (object): The credential object used to authenticate with cosmosdb. Default is None.
```

Comment:
hope we can align the docstring style in this pr...

Reply:
Refer doc: https://github.com/microsoft/promptflow/blob/main/docs/dev/documentation_guidelines.md#write-class-docstring

Reply:
Fixed. Thanks.